src/components/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -144,6 +146,8 @@ const emit = defineEmits<{
   (e: 'change', value: DropdownItem): void;
 }>()
 
+const hidePopover = ref<true | undefined>()
```

Comment:
question: why would we type to `true | undefined`? If it's not set to true, it's going to eval the same as false in the `KPop` component right?

Reply:
I'm still not sure I understand. You're essentially creating what appears to be a boolean  behave like a non-boolean (non-native) type. I think a regular boolean would be more expected

Reply:
Think of it like this: boolean means `hidePopover.value` could be either `=== true` or `=== false`, meanwhile in our component we only want it to be `=== true` or undefined.

Reply:
I understand what you're saying; however, I don't understand why `false` is invalid here for either component.

Reply:
Changed to `boolean`